### DISCUSSION TITLE
i get judged for things i didn't do please help!!! IF I LOSE, THERE'S NOTHING I CAN DO I'M SO SORRY you are the best

### DISCUSSION THREAD

**Discussion by Derzweite77** [community/community#57619] [Other Feature Feedback, Questions, & Ideas]
Ich werde hir verurteilt und gerichtet ohne das ich eine Anhörung hatte , und weiß nicht einmal was ist passiert 

I'm being tried and judged here without even having a hearing and I don't even know what happened

**Comment by DSDmark** [comment]
Hi @Derzweite77,

Can you **specific more** about the details of what you want? For better understanding 😅. Also, you can add some **screenshots** or **error** which you were facing 👍.
